packages/apps/tools/TESTPLAN.md

```diff
@@ -0,0 +1,87 @@
+- **Inputs:** User enters a valid key and adds to the list on clicking '+' next to the input.
+- **Expected Outcome:** The user enters a valid public key and the account name input is automatically filled with a k:account. The user clicks on the button and after a successful creation and funding, sees a notification that the transfer was successful to the given account.
+
+#### Scenario 2: Get error on entering invalid public key (length validation)
+- **Objective:** User cannot create an account with an invalid public key and fund 100 coins to it.
+- **Preconditions:** /
+- **Inputs:** User enters an invalid key and adds to the list on clicking '+' next to the input.
+- **Expected Outcome:** The user enters an invalid public key and the input shows error status. If the user clicks on the submit button sees a error text to provide valid public key(s) and cannot fund the given account.
+
+#### Scenario 3: Create multi key w:account and successfully fund 100 coins
+- **Objective:** User can create an account with multiple valid public keys and fund 100 coins to it.
+- **Preconditions:** User has two or more public keys already created.
+- **Inputs:** User enters valid keys and adds them to the list on clicking '+' next to the input.
+- **Expected Outcome:** The user enters multiple valid public keys and the account name input is automatically filled with a w:account. The user clicks on the button and after a successful creation and funding, sees a notification that the transfer was successful to the given account.
+
+#### Scenario 4: Get notification when error happens during transfer
+- **Objective:** User gets notification with error status that includes the error message if the transfer is unsuccessful.
+- **Preconditions:** User has one or more public keys already created.
+- **Inputs:** User enters (one or more) public key(s) and adds to the list on clicking '+' next to the input. The account name is auto generated. The user selects a chain.
+- **Expected Outcome:** The user enters (one or more) public key(s) and selects chain to which an account will be created and funds will be added. An error happens and the user sees a notification displaying the error message.
+
+
+### 3.3 [Faucet / Fund existing accounts]
+
+#### Scenario 1: Successfully fund 100 coins to an existing account
+- **Objective:** User can fund 100 coins to an existing account.
+- **Preconditions:** User has an account already created.
+- **Inputs:** User enters a valid account name.
+- **Expected Outcome:** The user enters a valid account name. The user clicks on the submit button and after a successful funding, sees a notification that the transfer was successful to the given account.
+
+#### Scenario 2: Get error on entering invalid account name (length validation)
+- **Objective:** User cannot fund 100 coins to an invalid account name.
+- **Preconditions:** User has an account already created.
+- **Inputs:** User enters an invalid account name.
```

Comment:
Should we specify how to achieve this? In this case; entering <3 or >256 characters?